source/mainnet/net/guides/run-node-macos.rst

```diff
@@ -258,6 +259,31 @@ If you want to change whether the node services start automatically, you have tw
 
          $sudo rm "/Library/LaunchDaemons/software.concordium.testnet.node-collector.plist"
 
+.. _node-collector-configuration-macos:
+
+Node collector configuration
```

Comment:
We should make sure to note this migration in the release notes @dg-concordium

Reply:
I've added links in the release note per platform in the testnet release PR.